### PR TITLE
[BENCH-3845] Rename ConceptSet to FeatureSet in backend and backend APIs

### PR DESCRIPTION
**Pull request by dexamundsen**
UI already uses `feature set`. This PR renames ConceptSet to FeatureSet in backend and backend APIs
No change to the DB or DB fields. Leaving it as-in for now in DB

----

Plan is to get it reviewed internally before sharing with larger VUMC team

### CODE REVIEW

File: ui/src/data/source.tsx [hunk 1]
```diff
@@ -250,12 +250,6 @@ export type Criteria = {
   predefinedDisplayName?: string;
 };
 
-export type ConceptSet = {
```
Comment (dexamundsen) [inline ui/src/data/source.tsx:253]: unused

**Review by tjennison-work — APPROVED** [review]

**Review by marikomedlock — APPROVED** [review]
Thanks for all the updates!

Please give the VUMC teams (SD/eMERGE & AoU) a heads up about the OpenAPI changes because I think they will both need to make sure their other service (VUMC admin & AoU-RW) is running with a compatible version of the Tanagra client library before upgrading to the most recent Tanagra service.

**Comment by chenchals** [comment]
@dexamundsen The backed is still a disconnect if you rename to FeatureSet.  See https://github.com/DataBiosphere/tanagra/blob/d0e0f5fd534a1eb0aa467c4f6527d1c48de3562a/service/src/main/resources/db/changesets/20230922_initial_schema.yaml#L82-L148

**Comment by chenchals** [comment]
@dexamundsen The backed is still a disconnect if you rename to FeatureSet.  See https://github.com/DataBiosphere/tanagra/blob/d0e0f5fd534a1eb0aa467c4f6527d1c48de3562a/service/src/main/resources/db/changesets/20230922_initial_schema.yaml#L82-L148  This change will be a breaking change for tanagra-admin service

**Comment by dexamundsen** [comment]
> @dexamundsen The backed is still a disconnect if you rename to FeatureSet. See
> 
> https://github.com/DataBiosphere/tanagra/blob/d0e0f5fd534a1eb0aa467c4f6527d1c48de3562a/service/src/main/resources/db/changesets/20230922_initial_schema.yaml#L82-L148
> 
> This change will be a breaking change for tanagra-admin service

he DB table and field names were retained as concept_set (at least for now) to avoid more invasive changes.

**Comment by dexamundsen** [comment]
@chenchals please confirm that we can merge this PR

**Review by chenchals — COMMENTED** [review]
@dexamundsen  Sure please go ahead and merge these changes.

**Pull request merged**